### PR TITLE
add: Quiz Functionality

### PR DESCRIPTION
**Pull request by shilpe26**
Added Quiz Functionality with features like:-
▪️Categories of Quizzes
▪️Play quiz 
▪️Get Score

quiz functionality

### CODE REVIEW

File: src/stylesheets/warning.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    right: 0;
+    left: 0;
+    top: 0;
+    bottom: 0;
+    z-index: 9;
+    display: grid;
+    place-items: center;
+}
+.warning-modal{
+    background-color: #fff;
```
Comment (surajmore98) [inline src/stylesheets/warning.css:13]: you can create variable for this.

File: src/stylesheets/warning.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.warning-container{
+    background-color: rgba(0,0,0,0.3);
```
Comment (surajmore98) [inline src/stylesheets/warning.css:2]: you can create variable for this.

File: src/pages/result/Result.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useCategory } from "../../context/category-context";
+import "./result.css";
+function Result() {
+	const { getCategory, setGetCategory } = useCategory();
+
+	return (
+		<div className="text text-lg">
+			Your Final Score is{" "}
```
Comment (surajmore98) [inline src/pages/result/Result.jsx:9]: you can remove these empty parenthesis

**Review by surajmore98 — APPROVED** [review]

**Review by mdarif6 — APPROVED** [review]
- I like the naming convention
- small suggestions as earlier pr said- use variables will looks good instead direct name
- clean and structured code.

**Pull request merged**